File: programmers/Level2/기능개발.py
import math
def solution(progresses, speeds):
    answer = []
    k = 0
    standard = 0
    for i in range(len(progresses)):
        speeds[i] = math.ceil((100-progresses[i])/speeds[i])
    for i in range(len(speeds)):
        if standard != 0:
            if speeds[i] <= standard:
                answer[-1] += 1
            else:
                standard = 0
        if standard == 0:
            standard = speeds[i]
            answer.append(1)
    print(answer)
    return answer

File: programmers/stack_queue/기능개발.cpp
#include <vector>
#include <cmath>

using namespace std;

vector<int> solution(vector<int> progresses, vector<int> speeds) {
	vector<int> answer;
	int size = progresses.size();
	int idx = 0;
	int day = 0;

	while (idx < size) {
		day = (int)ceil((100 - progresses[idx]) / (double)speeds[idx]);
		answer.push_back(1);
		++idx;
		while (idx < size) {
			if (progresses[idx] + speeds[idx] * day >= 100) {
				++answer.back();
				++idx;
			}
			else {
				break;
			}
		}
	}
	return answer;
}
